dist/gulpfile.babel.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
```

Comment:
dist folder should also be ignored. The content of the dist folder and coverage folder can always and will always be generated